app/models/concerns/orchestration/compute.rb

```diff
@@ -37,7 +37,7 @@ def vm_name
 
   def queue_compute
     return unless compute? && errors.empty?
-    new_record? ? queue_compute_create : queue_compute_update
+    new_vm? ? queue_compute_create : queue_compute_update
```

Comment:
The idea behind this was the following:
For an API request: When a user passes `compute_resource_id` and `uuid` when creating a host, the user wants to associate an existing VM to Foreman.  So no new VM should be created.
For an UI request, `uuid` is always empty (and `new_vm?` false) except when using the import option.